### ISSUE TITLE
TypeError: Cannot read property 'some' of undefined

### ISSUE THREAD

**Issue by lanlin** [eddyerburgh/avoriaz-karma-mocha-example#1]
Hi,
Thanks for your great work.

I tried avoriaz + karma + mocha with the karma.conf.js in this example project, i got an error:

ERROR [config]: Invalid config file!
TypeError: Cannot read property 'some' of undefined
at Object.<anonymous> (..../test/unit/karma.conf.js:30:29)

I dont known how to resolve it.

**Comment by eddyerburgh** [comment]
Did you install all the karma dependencies?

```
npm install --save-dev karma karma-coverage karma-mocha karma-phantomjs-launcher karma-sinon-chai karma-sourcemap-loader karma-spec-reporter karma-webpack
```

This project is running the tests correctly, so if you use it as a base, it should work 👍

**Comment by lanlin** [comment]
![image](https://user-images.githubusercontent.com/3468024/26858787-fc908bca-4b65-11e7-8d4c-5dcae9fd9873.png)

All needed dependencies has been installed aready, but still have the same error.

![image](https://user-images.githubusercontent.com/3468024/26858883-aaa438d8-4b66-11e7-8baa-1958fa1d3181.png)

I change back to original karma.conf.js as vue-cli using, it works.
Thanks anyway.

**PS**: Is there a way to mock some functions (APIs) or components?

**Comment by eddyerburgh** [comment]
Yeah, check out - https://www.coding123.org/stub-dependencies-vue-unit-tests/

To stub methods you can use sinon - http://sinonjs.org/releases/v2.3.2/
